src/src/MainArea/MainArea.tsx

```diff
@@ -4,22 +4,23 @@ import Bookshelf from './Bookshelf/Bookshelf'
 import Archive from './Archive/Archive'
 
 interface IProps {
-    visiblePage: string;
+    activePage: string;
 }
 
 type eVisibleComponent = {
     bookShelf: JSX.Element;
     addBookForm: JSX.Element;
     archive: JSX.Element;
+    [key: string]: JSX.Element;
   }
 
-const MainArea = ({visiblePage}: IProps) => {
+const MainArea = ({activePage}: IProps) => {
     const visibleComponent: eVisibleComponent = {
         bookShelf: <Bookshelf />,
         addBookForm: <AddBookForm />,
         archive: <Archive />
     }
-    return (visibleComponent[visiblePage])
+    return (visibleComponent[activePage])
```

Comment:
If you're changing the same code in the same Pull Request it makes sense to optimize the review (and the history lookup) process. The way to do that is through rebasing. Remind me to discuss it.